app-doc/audacity/audacity-9999.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="HTML reference manual for Audacity"
+HOMEPAGE="https://www.audacityteam.org/"
+
+if [[ ${PV} = 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/audacity/audacity-manual.git"
+else
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/help"
+	SRC_URI="amd64? ( https://github.com/audacity/audacity-manual/releases/download/v${PV}/audacity-manual-${PV}.tar.gz )"
```

Comment:
I wonder if this part shouldn't be unconditional. Is anything about it amd64-specific?

(That doesn't mean we should add keywords for other arches until it's been tested, but I'm wondering if the distfile is actually amd64-specific.)

Reply:
After a quick grep operation over the manual, I can't find anything architecture specific except that [Audacity will not run on the Windows RT operating system which ships on devices running ARM architecture](https://manual.audacityteam.org/man/faq_about_audacity.html#mobile).